ios/brave-ios/Sources/AIChat/Resources/tr.lproj/BraveLeo.strings

```diff
@@ -0,0 +1,291 @@
+
+/* The title for view which user type feedback */
+"aichat.feedbackInputViewTitle" = "Geri bildirimi buraya girin";
+
+/* The title for view which listsfeedback option list */
+"aichat.feedbackOptionsViewTitle" = "Geri bildiriminiz neyle alakalı?";
+
+/* The title for helpful feedback option */
+"aichat.feedbackOptionTitleNotHelpful" = "Yanıt yardımcı olmadı";
+
+/* The title for not working feedback option */
+"aichat.feedbackOptionTitleNotWorking" = "Bir şey çalışmıyor";
+
+/* The title for other feedback option */
+"aichat.feedbackOptionTitleOther" = "Diğer";
+
+/* The title for premium ad view. The links are inside parenthesis and the phrases inside square brackets should be translated */
+"aichat.feedbackPremiumAdTitle" = "Leo Premium, daha yüksek yanıt nüansı için genişletilmiş dil modellerine erişim sunmaktadır. [Daha fazla bilgi](https://brave.com/) - [Kapat](braveai://dismiss)";
+
+/* The title for the button that submits feedback */
+"aichat.feedbackSubmitActionTitle" = "Gönder";
+
+/* The title for feedback view when it fails to submit feedback */
+"aichat.feedbackSubmittedErrorTitle" = "Bir şeyler ters gitti. Lütfen yeniden deneyin.";
+
+/* The title for feedback view when it is submitted */
+"aichat.feedbackSubmittedTitle" = "Geri bildirim başarıyla gönderildi";
+
+/* The title for feedback view when response is sucessfull but disliked */
+"aichat.feedbackSuccessAnswerDisLikedTitle" = "Yanıt beğenilmedi";
+
+/* The title for feedback view when response is sucessfull also liked */
+"aichat.feedbackSuccessAnswerLikedTitle" = "Yanıt beğenildi";
+
+/* The title for view which user type feedback */
+"aichat.feedbackViewMainTitle" = "Brave YZ Geri Bildirimi Verin";
+
+/* The button title for opening paywall screen */
+"aichat.goPremiumButtonTitle" = "Premium'a Geçin";
+
+/* Description about answers are changing with page context */
+"aichat.infoAboutPageContext" = "Yanıtları, sayfanın içeriklerine bağlı olarak şekillendirin";
+
+/* The model intro message when you first enter the chat assistant */
+"aichat.introMessageClaudeInstantMessageDescription" = "Merhaba, ben Leo. Brave tarafından vekaleten kullanılıyorum ve Anthropic tarafından diyalog ve metin işleme görevlerini yerine getirmesi için tasarlanan Claude Instant ile destekleniyorum.";
+
+/* The model and creator for intro message - Claude Instant is the model -- Anthropic is the creator */
+"aichat.introMessageClaudeInstantModelDescription" = "Anthropic tarafından geliştirilen Claude Instant";
+
+/* The model intro message when you first enter the chat assistant -- %@ is a place-holder for the model name */
+"aichat.introMessageGenericMessageDescription" = "Merhaba, ben Leo. Brave tarafından geliştirilmiş bir YZ asistanıyım. %@ tarafından destekleniyorum. Bana istediğinizi sorun, yanıtlamak için elimden geleni yapacağım.";
```

Comment:
"Merhaba, ben Leo. Brave tarafından geliştirilmiş bir Yapay Zeka asistanıyım. %@ tarafından destekleniyorum. Bana istediğinizi sorun, yanıtlamak için elimden geleni yapacağım."